massadmin/massadmin.py

```diff
@@ -387,7 +387,16 @@ def mass_change_view(
             obj=obj)
 
 
-class MassEditMixin:
-    actions = (
-        mass_change_selected,
-    )
+class MassEditMixin(object):
+
+    def get_actions(self, request):
+        actions = super(MassEditMixin, self).get_actions(request)
```

Comment:
Couldn't we the Python 3+ style `super().get_actions(request)`?

Reply:
yess. done.

Reply:
basically the suggested code keep the mass edit action hidden from admin action list. Core app code is untouched.
I don't know how to test this. Suggestions are wellcome.